laboratory/minje/data_save_pipline.py

```
@@ -0,0 +1,138 @@
+    # --------------------------------------------------------------------------
+    print(">>> [System] MCP 도구 로드 중...")
+    try:
+        mcp_tools = await client.get_tools()
+
+        # update_corplist 선행 실행
+        update_tool = next((t for t in mcp_tools if t.name == "update_corplist"), None)
+        if update_tool:
+            print(">>> [System] DART 기업 코드 파일 갱신 중... (1회)")
+            await update_tool.ainvoke({})
+            print(">>> [System] 갱신 완료.")
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)